Project.toml

```
@@ -12,9 +12,16 @@ Meshes = "eacbb407-ea5a-433e-ab97-5258b1ca43fa"
 QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
+[weakdeps]
+Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
+
+[extensions]
+EnzymeExt = "Enzyme"
```

```suggestion
MeshIntegralsEnzymeExt = "Enzyme"
```
I think the naming convention for extensions is to also include the name of the base package. Of course this also requires to update the filename `EnzymeExt.jl` to `MeshIntegralsEnzymeExt.jl` and the name of the module.